Patch taken from the libinfinity git repository to make a test using
certificates succeed.

From 859b98ec6588a8e3ae91b6f6f4753dc7c5c4c7f2 Mon Sep 17 00:00:00 2001
From: Philipp Kern <phil@philkern.de>
Date: Sun, 26 May 2024 19:56:30 +0200
Subject: [PATCH] test/certs/ca-crt.pem: Change the expiry a long time into
 future.

---
 test/certs/ca-crt.pem | 64 +++++++++++++++++++++----------------------
 1 file changed, 32 insertions(+), 32 deletions(-)

diff --git a/test/certs/ca-crt.pem b/test/certs/ca-crt.pem
index cdbe855..5d2b661 100644
--- a/test/certs/ca-crt.pem
+++ b/test/certs/ca-crt.pem
@@ -1,34 +1,34 @@
 -----BEGIN CERTIFICATE-----
-MIIF1TCCA72gAwIBAgIJANHP24YS11wSMA0GCSqGSIb3DQEBCwUAMIGAMQswCQYD
-VQQGEwJVUzEWMBQGA1UECAwNTWFzc2FjaHVzZXR0czEQMA4GA1UEBwwHQmVsbW9u
-dDEOMAwGA1UECgwFMHg1MzkxFzAVBgNVBAMMDmdvYmJ5LjB4NTM5LmRlMR4wHAYJ
-KoZIhvcNAQkBFg9hcm1pbkBhcmJ1ci5uZXQwHhcNMTUwNTEwMjEzNTQzWhcNMjUw
-NTA3MjEzNTQzWjCBgDELMAkGA1UEBhMCVVMxFjAUBgNVBAgMDU1hc3NhY2h1c2V0
-dHMxEDAOBgNVBAcMB0JlbG1vbnQxDjAMBgNVBAoMBTB4NTM5MRcwFQYDVQQDDA5n
-b2JieS4weDUzOS5kZTEeMBwGCSqGSIb3DQEJARYPYXJtaW5AYXJidXIubmV0MIIC
-IjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA+Xc2ONjb1wOVibJqWvaruN6d
-m8hJWBY0T5R9rtnhRki2lSvgDIfCEEMDtfumuWimKMi27X0OKIzEFnZHAbtr9J8H
-CWTIxNac7a9LMomNEPu8MM8pYsFLgwN8SNDgZi0OXuc6Ox81D0iT6xlNi1uLiaRR
-PjIMxlsF7oM4OL1QZkL9FChcsIO8Xx7lg0FgfBlY4UFCoFBQDYQDIfXZKmPG3uh+
-I07uB20hEv7ki4kgvusZmJgfONB/meBt4rwH6XkrxR69phdph7SOB+CiE9H/TOup
-0wrOl6nzmYsp4+j3GrLW3iayEjfYtz12fmyBatdHElxVNOJZ99rFpnhLHBnc0B7B
-Z4pDj41lkytd47gGV6azLxVym6IBzO+RsQQ7AhSznKnfnAyw4w5ryMjVLBOXJOzW
-wtchaD55lQLjveMosiFc/iplehFyFkCadENw+ttjnXqwfHHbAFB6GVzDKUa2I3XY
-NW5Ec+WP3ee7y3KBHuflMXUL9FaHjVUlp/bKyiSNak9Ky1bXYTR7XUzrrqqY3tI2
-qT5XkJISgfbVTfWAa0am2jItj7wfsw0rjmD7G6cFTLxj3do13C8mlV758Jn99kpx
-ArSZjN5enPFRXD0ROJq7XGFlriQaHO+S2ehZauGY2wRKIYiy7FB+TrqMAfmc5/bI
-U6kGVpOWP0hU4ZkWwQ0CAwEAAaNQME4wHQYDVR0OBBYEFAtpZdHL1tG5lgY1pebQ
-HV9BqRtEMB8GA1UdIwQYMBaAFAtpZdHL1tG5lgY1pebQHV9BqRtEMAwGA1UdEwQF
-MAMBAf8wDQYJKoZIhvcNAQELBQADggIBAK8C9a/73E9KsFjhmbMRHyFQh0DqtC9R
-1tketMDNkYXuYuCPb/Oyl4vtWcADA3DnEv8P1RZqt2w6Ke58cJJ/YFBWrgxDeaIa
-UFdRRwzWYwzBRFUFVBjb2G00PdosQi55lsizxLgZVeH9k745ZIGx18oo6nEXHNZ8
-Ig0YKHwTVSS0eJ55RzR+CwaGwDXvezMNPVqPXtKOt39WLDyjlqSfi0naMNTyyJW1
-uSZIc9AH/xu/okoJUt+dllrr/j0k01JgS16N/dquaaXGXAzrszAnG4YWjPgySik+
-zxnQYBXpL+bzCGtB/0CxyajaANg4Q8n6CCr0fQ97DgRfBSLlH1MUHzK2zSdVDq+b
-NV+CtJ2U94AigaVDgfFCKZGRxZklnz56QL37oawi3yRh/Ba7GGCEplgYQc3pjGqc
-Wd+zD2e9vp5LI1POHBCeAp/4j3Kgx/uOCpiXJfBVfkQwZ032I1E1DzrJkYEuXg/Q
-+07MWPxiJLBqr0ROgq2pj3i5bwIIaf/yubeQG3ftFmW2AcovZZTFz6QQr789+EgY
-K5u4dT/rQIOhbWZuDfP3/RCE727xCjheGpNM6d6458mf33u6wvo0aRUjl3fIhgJ2
-M3X5hm3SjorN2oeCTXVDIMazjfPZQYAcKgSWhn/UjLK87vnECk5ABuxb8ycAcEd2
-HM48UDyzAu7J
+MIIF4zCCA8ugAwIBAgIUEIgsZVRnN/fwe6ihYsIyY6EPaSswDQYJKoZIhvcNAQEL
+BQAwgYAxCzAJBgNVBAYTAlVTMRYwFAYDVQQIDA1NYXNzYWNodXNldHRzMRAwDgYD
+VQQHDAdCZWxtb250MQ4wDAYDVQQKDAUweDUzOTEXMBUGA1UEAwwOZ29iYnkuMHg1
+MzkuZGUxHjAcBgkqhkiG9w0BCQEWD2FybWluQGFyYnVyLm5ldDAeFw0yNDA1MjYx
+NzU1NTBaFw00NDA1MjExNzU1NTBaMIGAMQswCQYDVQQGEwJVUzEWMBQGA1UECAwN
+TWFzc2FjaHVzZXR0czEQMA4GA1UEBwwHQmVsbW9udDEOMAwGA1UECgwFMHg1Mzkx
+FzAVBgNVBAMMDmdvYmJ5LjB4NTM5LmRlMR4wHAYJKoZIhvcNAQkBFg9hcm1pbkBh
+cmJ1ci5uZXQwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQD5dzY42NvX
+A5WJsmpa9qu43p2byElYFjRPlH2u2eFGSLaVK+AMh8IQQwO1+6a5aKYoyLbtfQ4o
+jMQWdkcBu2v0nwcJZMjE1pztr0syiY0Q+7wwzyliwUuDA3xI0OBmLQ5e5zo7HzUP
+SJPrGU2LW4uJpFE+MgzGWwXugzg4vVBmQv0UKFywg7xfHuWDQWB8GVjhQUKgUFAN
+hAMh9dkqY8be6H4jTu4HbSES/uSLiSC+6xmYmB840H+Z4G3ivAfpeSvFHr2mF2mH
+tI4H4KIT0f9M66nTCs6XqfOZiynj6PcastbeJrISN9i3PXZ+bIFq10cSXFU04ln3
+2sWmeEscGdzQHsFnikOPjWWTK13juAZXprMvFXKbogHM75GxBDsCFLOcqd+cDLDj
+DmvIyNUsE5ck7NbC1yFoPnmVAuO94yiyIVz+KmV6EXIWQJp0Q3D622OderB8cdsA
+UHoZXMMpRrYjddg1bkRz5Y/d57vLcoEe5+UxdQv0VoeNVSWn9srKJI1qT0rLVtdh
+NHtdTOuuqpje0japPleQkhKB9tVN9YBrRqbaMi2PvB+zDSuOYPsbpwVMvGPd2jXc
+LyaVXvnwmf32SnECtJmM3l6c8VFcPRE4mrtcYWWuJBoc75LZ6Flq4ZjbBEohiLLs
+UH5OuowB+Zzn9shTqQZWk5Y/SFThmRbBDQIDAQABo1MwUTAdBgNVHQ4EFgQUC2ll
+0cvW0bmWBjWl5tAdX0GpG0QwHwYDVR0jBBgwFoAUC2ll0cvW0bmWBjWl5tAdX0Gp
+G0QwDwYDVR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAgEA3n++Xbrfypm3
+s2u2tTNHK+rH2VYZqoyjRShCNleG1OkZrAhhmDK2oDnolJIsx3ajmT3IwovamtLT
+kbwO+brv6g9IdDTWDXS4PKcWmZNQEPrul7ges3Dl6l0UL0w0fzrVWAQ4iUHGD9n6
+nJx7DNC/VG+GblR+9Dt2tqeQMEIzan6rMlbqYNFrwByL+HMPusIMvxRPdfnVs1nn
+TVqS9TrIlQiZr6dX/hQ3mTbvRRT5LrAexAeIg38bCSqC5MigvEMd0C3znWfDAJiJ
+jXHnJWnp1EsZ6AJYIdejooxRrJHZ7TjFe8msScy1lqS5WSvBNxwdEIUvStU0Vkro
+pnH8X0Mp+PoUyjqG9LpY+NnvOG4brTy92TVe4g5eG7n2edB/z11hiKpo2Fr5XcDy
+cVKvI7f86e172XQtvZoWAmpxyJlbLLx7uUCUEPcvOkdjhwLedBoczDOu9QsMfFT2
+3vrib2pWH+meUU++Q4PRyOeefqfQAGwpfAGW2aaMk8TNyx9Aldf68x6cNECr07ju
+tlVZbz5M1LMSAiD4xtGMYOPFPkCNPnwsYY2fKyg6DStlpLz6p4snO6EMIe4wJgNl
+/T3CcBTOBglR08kz430VLzwHiDb7FBb+Cai+LgRKyvva1WGr46aeASiQMBMDPAbT
+61dQEcth999mrT8ZhjYphL1gklNPCs8=
 -----END CERTIFICATE-----
-- 
2.49.0

